frontend/src/pages/hardwareProfiles/utils.ts

```diff
@@ -152,12 +152,10 @@ export const validateProfileWarning = (
   const warnings = hardwareProfileWarningSchema.safeParse({ isDefault: false, value: identifiers });
   if (warnings.error) {
     warnings.error.issues.forEach((issue) => {
-      if ('params' in issue && typeof issue.params !== 'undefined') {
-        warningMessages.push({
-          message: issue.message,
-          type: issue.params.type,
-        });
-      }
+      warningMessages.push({
+        message: issue.message,
+        type: HardwareProfileWarningType.OTHER,
+      });
```

Comment:
I think we can get rid of this HardwareProfileWarning in favor of having custom typed zod errors. we should be able to set a. custom error type in the schema. doing that would make this function simply do a safe parse of the whole hardware profile

Reply:
you are right. i dont think we can add it there. I was thinking of a user case we have that looks like this. We are fine then

```tsx
 code: z.ZodIssueCode.custom,
params: { code: ValidationErrorCodes.DUPLICATE },
```

Reply:
however we dont need to have a custom type called other. we should chnage it to be more specific on what type of error it i